17thblog/style.css

```diff
@@ -0,0 +1,76 @@
+h1 {
+	text-align: center;
+	color: #061643;
+	padding: 10px;
+	margin: 10px;
+	background-color: white;
+
+}
+
+h2 {
+	font-weight: 400;
+	text-align: left;
+	border-bottom: dashed 1px #051138;
+	color: #061643;
+}
+
+.navbarStyle {
+	text-align: center;
+	background-color: white;
+	font-size: 25px;
+	border-radius: 25px;
+	padding:10px;
+}
+
+
+a:hover{
+	 background-color: #e6e2ff;
+}
+
+.textStyle {
+	background-color: white;
+	margin:50px;
+	margin-bottom:50px;
+	border-radius: 15px;
+	padding:10px;
+}
+
+
+.centerImage {
+	display: block;
+	margin: auto;
+	width: 100%;
+	z-index: -1;
+	position: relative;
+}
+
+/*background */
+.bg {
+	background: linear-gradient(to bottom, #b2f7ff 0%, #b8b1ef 100%);
+	/*
+	font-family: "Verdana", Verdana, sans-serif;
+	*/
+	font-family: "Source Sans Pro", Helvetica, sans-serif;
+	font-weight: 300;
+	line-height: 1.5;
+}
+
+/* this holds the image and text*/
+.container {
+	position: relative;
+    text-align: center;
```

Comment:
Not  a big deal, but some of the indentation on this page is spaces, and some is tabs. If you select everything inside the curly braces and hit `shift + tab` it will un-indent everything, and then hit tab to re-indent it all as tabs.

Reply:
Will do! Thanks! (Also sorry that I messed up the PR both times, think I finally get it now but will probably grab you sometime to check too. Thanks!)